docs/build/configure/triggers.md

```diff
@@ -209,32 +209,79 @@ The following attributes are available for triggers:
 5. Write your cloud function or lambda to process the request from `viam-server`.
    The following example function prints the received headers:
 
-   ```python {class="line-numbers linkable-line-numbers"}
-   import functions_framework
-   import requests
-   import time
+   {{< tabs >}}
+   {{% tab name="Flask" %}}
+
+```python {class="line-numbers linkable-line-numbers" }
+from flask import Flask, request
+
+app = Flask(__name__)
+
+
+@app.route("/", methods=['GET', 'POST'])
+def hello_http_get():
+    headers = request.headers
+    print(headers)
+    payload = {
+        "Org-Id": headers.get('org-id', 'no value'),
+        "Organization-Name": headers.get('organization-name', 'no value'),
+        "Location-Id": headers.get('location-id', 'no value'),
+        "Location-Name": headers.get('location-name', 'no value'),
+        "Part-Id": headers.get('part-id', 'no value'),
+        "Part-Name": headers.get('part-name', 'no value'),
+        "Robot-Id": headers.get('robot-id', 'no value'),
+        "Machine-Name": headers.get('machine-name', 'no value'),
+        "Component-Type": headers.get('component-type', 'no value'),
+        "Component-Name": headers.get('component-name', 'no value'),
+        "Method-Name": headers.get('method-name', 'no value'),
+        "Min-Time-Received": headers.get('min-time-received', 'no value'),
+        "Max-Time-Received": headers.get('max-time-received', 'no value'),
+        "Data-Type": request.args.get('data_type', 'no value')
```

Comment:
ComponentName, ComponentType, MethodName, MachineName, LocationName, OrgName, FileID, FilePath are each parsed from within the request's body, not headers